crt_portal/cts_forms/templates/forms/widgets/multi_select.html

```diff
@@ -1,12 +1,22 @@
-<select class="complaint-multi-select {{widget.attrs.class}}" multiple name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %}>
```

Comment:
Isn't this file part of Haseeb's work. May run into merge conflicts, check with him please.